shared/src/components/exercise-preview/index.jsx

```diff
@@ -63,22 +63,23 @@ class ExercisePreview extends React.Component {
   };
 
   getCleanPreview = () => {
-    return trimEnd(last(this.exercise.questions).stem_html);
+    return this.props.extractedInfo.preview ? this.props.extractedInfo.preview : trimEnd(last(this.exercise.questions).stem_html);
   };
```

Comment:
this is the really crazy bit.  the `getCleanPreview` method wasn't even reading the preview

Reply:
I was wondering, in a MPQ without video or sim, would it always show the last question stem at the top?

Reply:
hm, good point.  yes that does seem like it would.  I'd just copied the existing code, but I agree that ti'd make more sense to pick the first part